src/mongo/util/net/sockaddr.cpp

```diff
@@ -144,7 +144,7 @@ bool SockAddr::isLocalHost() const {
         default:
             return false;
     }
-    fassert(16502, false);
+    //fassert(16502, false);
```

Comment:
If you want to remove code, remove it. Even better here would be to write MONGO_UNREACHABLE, in which case you can probably remove the ```return false``` as well, since MONGO_UNREACHABLE has ```noreturn``` semantics.